### ISSUE TITLE
Ability to see how many people are in the chat

### ISSUE THREAD

**Issue by Soyeon-merz** [nextcloud/spreed#5523]
Right next to “Participants” could be shown the number of participants and how many are online.

![headcount](https://user-images.githubusercontent.com/83290151/116280654-349d1500-a789-11eb-9905-f0d453002de5.jpg)

**Comment by PVince81** [comment]
that could be nice, yes.

also optionally how many are currently in a call, but could quickly become visually cluttering

**Comment by nickvergessen** [comment]
Yeah, I don't see a (5/12/20) thing there. Some countries might even think it's a date

**Comment by Soyeon-merz** [comment]
What about just "how many are online?" (20)

**Comment by Soyeon-merz** [comment]
For example "Microsoft Teams":
![count](https://user-images.githubusercontent.com/83290151/116539317-f454a880-a8e8-11eb-9265-835984fff336.jpg)

**Comment by Soyeon-merz** [comment]
![headcount_compare](https://user-images.githubusercontent.com/83290151/116627104-cc972c00-a94c-11eb-9315-501226abe9f1.jpg)

**Comment by nickvergessen** [comment]
> What about just how many are online?
even without “Participants”?

I think showing one number is fine, maybe even (X/Y), just not 3 numbers, as then it becomes unclear what it means.

But as said we already talked about briefly yesterday, if the call gets it's details in the topbar, that could show the call participants there, so showing the number of "chatting" people in the sidebar should be fine.
